Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: speedhack toggle cheat

### Description (*)

This PR adds a cheat to drastically increase the player character's movement speed. Pressing the button toggles this mode.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2210](https://jira.unity3d.com/browse/MTT-2210)
### Manual testing scenarios

1. Start a game and progress to BossRoom scene
2. Open the cheat window and use the "Toggle Super Speed" button
3. Your character should now be much faster
### Questions or comments

### Contribution checklist
